src/bridge/views/SentTransaction.js

```diff
@@ -29,10 +33,16 @@ const Success = (props) => {
         className={'mb-4 mt-1'}
         prop-size={'sm'}
         target={'_blank'}
-        href={`https://${esdomain}/tx/${props.hash}`}>
+        href={`https://${esdomain}/tx/${hash}`}>
           {'View on Etherscan ↗'}
       </Anchor>
 
+  const confirmations = Maybe.fromNullable(txnConfirmations[hash]).getOrElse(0)
+
+  const requiredConfirmations = 1
+
+  const status = confirmations < requiredConfirmations ? 'Pending...' : `Confirmed x${confirmations}!`
```

Comment:
Would love to see a spinner displayed alongside the "Pending..." text, to indicate to the user that this is actively happening/being checked for.

PR looks good at first glance, I'll give this a proper test run & review tomorrow! (: